lib/cadet_web/admin_controllers/admin_grading_controller.ex

```diff
@@ -129,16 +122,9 @@ defmodule CadetWeb.GradingController do
   end
 
   def grading_summary(conn, _params) do
-    user = conn.assigns[:current_user]
-
-    case Assessments.get_group_grading_summary(user) do
+    case Assessments.get_group_grading_summary() do
```

Comment:
Example where user role validation is moved out of the Model to the router.

We lose a descriptive error message, and get a 403 Forbidden when unauthorized users access these endpoints.